cognite/seismic/protos/v1/seismic_service.proto

```diff
@@ -134,5 +134,5 @@ service SeismicAPI {
      * Returns a stream of file objects, terminating all files matching the search criteria have been
      * returned.
      */
-    rpc SearchFiles(SearchFilesRequest) returns (stream File) {}
+    rpc SearchFiles(SearchFilesRequest) returns (stream SourceSegyFile) {}
```

Comment:
Should we rename the search api too? SearchSourceFiles?

OTT, this looks good to me. =)